static/js/models/thing-model.js

```diff
@@ -216,7 +217,7 @@ class ThingModel extends Model {
     if (typeof this.propertiesHref === 'undefined') {
       const urls = Object.values(this.propertyDescriptions).map((v) => {
         if (v.forms) {
-          return v.forms[0].href;
+          return Utils.selectFormHref(v.forms, Constants.WoTOperation.WRITE_PROPERTY);
```

Comment:
I think for extensibility it might be right to look for `writeproperty` operation here even if now it is just reduntat.